plugins/@grouparoo/prometheus/package-lock.json

```diff
@@ -0,0 +1,10118 @@
+{
```

Comment:
If you are using `pnpm`, there shouldn't be a `package-lock.json` - this might explain some of the problems you were seeing installing things